lib/screens/SearchPage.dart

```diff
@@ -35,33 +35,29 @@ class _SearchPageState extends State<SearchPage> {
   Widget build(BuildContext context) {
     return Scaffold(
       appBar: AppBar(
-          surfaceTintColor: Colors.transparent,
-          toolbarHeight: 50 + $styles.padding.m,
-          title: Padding(
-            padding: EdgeInsets.symmetric(
-                vertical: $styles.padding.m, horizontal: $styles.padding.xs),
-            child: Search(
-              filterSearchResults: filterSearchResults,
-              editingController: editingController,
-            ),
-          )),
+        surfaceTintColor: Colors.transparent,
+        toolbarHeight: 50 + $styles.padding.m,
+        title: Padding(
+          padding: EdgeInsets.symmetric(
+            vertical: $styles.padding.m,
+            horizontal: $styles.padding.xs,
+          ),
+          child: Search(
+            filterSearchResults: filterSearchResults,
+            editingController: editingController,
+          ),
+        ),
+      ),
       body: Padding(
         padding: EdgeInsets.symmetric(horizontal: $styles.padding.l),
-        child: Column(
-          children: [
-            Expanded(
-              child: ListView.builder(
-                shrinkWrap: true,
```

Comment:
Column, Expanded and shrinkwrap aren't necessary. The ListView already expands to fill the available space.